### PR TITLE
Create bot.zh-tw.yml

### PR DESCRIPTION
**Pull request by danielpclin**
translation for mandarin - Taiwan

### CODE REVIEW

**Comment by GabrielRF** [comment]
@danielpclin Hello! Thank you for the translation! Can you talk to me on Telegram, please?
I'm not sure mandarin is supported by Telegram

**Comment by GabrielRF** [comment]
@danielpclin  https://t.me/GabrielRF

File: i18n/bot.zh-tw.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+zh-tw:
```
Comment (GabrielRF) [inline i18n/bot.zh-tw.yml]: I guess it should be `zh-hant-tw` as tested on Telegram. This should also be applied to the file name.

**Comment by danielpclin** [comment]
Is it case sensitive? @GabrielRF

**Comment by GabrielRF** [comment]
> Is it case sensitive? @GabrielRF

On my previous tests it was not. 

I'll accept the PR and then we test it.

Thank you very much!

**Pull request merged**